tests/perf-test/SidebarUtils.perf-test.ts

```diff
@@ -0,0 +1,92 @@
+import Onyx from 'react-native-onyx';
+import {measureFunction} from 'reassure';
+import SidebarUtils from '@libs/SidebarUtils';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import Policy from '@src/types/onyx/Policy';
+import Report from '@src/types/onyx/Report';
+import ReportAction from '@src/types/onyx/ReportAction';
+import createCollection from '../utils/collections/createCollection';
+import createRandomPolicy from '../utils/collections/policies';
+import createRandomReportAction from '../utils/collections/reportActions';
+import createRandomReport from '../utils/collections/reports';
+import * as LHNTestUtils from '../utils/LHNTestUtils';
+import waitForBatchedUpdates from '../utils/waitForBatchedUpdates';
+
+beforeAll(() =>
+    Onyx.init({
+        keys: ONYXKEYS,
+        safeEvictionKeys: [ONYXKEYS.COLLECTION.REPORT_ACTIONS],
+    }),
+);
+
+// Clear out Onyx after each test so that each test starts with a clean slate
+afterEach(() => {
+    Onyx.clear();
+});
+
+const getMockedReports = (length = 500) =>
+    createCollection<Report>(
+        (item) => `${ONYXKEYS.COLLECTION.REPORT}${item.reportID}`,
+        (index) => createRandomReport(index),
+        length,
+    );
+
+const reportActions = createCollection<ReportAction>(
+    (item) => `${item.reportActionID}`,
+    (index) => createRandomReportAction(index),
+);
+
+const mockedResponseMap: Partial<Report> = getMockedReports(5000);
```

Comment:
Hey, are you sure the type here is correct? I got `Record<string, Report>` based on infer (also, the type shouldn't have to be manually provided i think).

Reply:
You’re right. The type was incorrect. But still we need to cast type here as  data passed to `Onyx.multiset()` should conform type `Partial<NullableKeyValueMapping>` (https://github.com/Expensify/react-native-onyx/blob/d76bed7cd6b69e6a345956cb304dc0cf32620868/lib/Onyx.d.ts#L10)